src/scripts/entrypoint.sh

```diff
@@ -35,6 +35,13 @@ while [ true ]; do
     sleep 604810 &
     SLEEP_PID=$!
 
-    # Wait on sleep so that when we get ctrl-c'ed it kills everything due to our trap
-    wait "$SLEEP_PID"
+    # Wait for 1 week sleep or nginx
+    wait -n $SLEEP_PID $NGINX_PID
```

Comment:
I don't think this will work; this doesn't wait for `$SLEEP_PID` or `$NGINX_PID`, it waits for `$SLEEP_PID` _and_ `$NGINX_PID`.

Perhaps what we can do is instead of waiting for an entire week, we wait in 5s increments, then at the end of each second, we check to make sure that `nginx` is still alive, and if it is, we sleep for another `5s`, etc... and once a week's worth of sleeping has elapsed, we run `run_certbot.sh` again and reset our internal sleep counter.

Reply:
Well it does work. `wait -n` waits for any of the processes.

You can test this sample script. It simulates nginx failure and stops as exepected after 5 seconds.
```bash
#!/bin/bash
set -x

(nginx -g "daemon off;") &
export NGINX_PID=$!

# Sleep for 1 week
sleep 604810 &
SLEEP_PID=$!

# kill nginx after 5 seconds
(sleep 5; kill ${NGINX_PID}) &

wait -n $SLEEP_PID $NGINX_PID
```

Reply:
I stand corrected!  The `-n` option didn't show up in my man page for `wait`.  :)